### PR TITLE
Sf3 compatibility fix for refresh-metadata and update-cdn-status commands

### PR DESCRIPTION
**Pull request by vyshkant**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Fixed
- Sf3 compatibility on the refresh-metadata and update-cdn-status commands (dialog helper)
```

## Subject

Calls of DialogHelper replaced with calls to QuestionHelper

### CODE REVIEW

**Comment by greg0ire** [comment]
Please format your commit message according to our rules : 

- The commit message subject must be less than 50 characters and tell us *what* you did.
- The commit message body should tell us *why* you did it. It is optional but highly recommended.

Bad example :

```
Fixed bug #989 by removing call to defraculate()
```

Good example:

```
Remove call to defraculate()

Calling this function caused a bug because it interferes 
with calls to getSchmeckles().
Fixes #989
```

More details in CONTRIBUTING.md

File: Command/RefreshMetadataCommand.php [hunk 1]
```diff
@@ -51,33 +59,29 @@ public function configure()
...
+
         $medias = $this->getMediaManager()->findBy([
-            'providerName' => $provider,
+            'providerName' => $provider->getName(),
             'context' => $context,
         ]);
 
-        $this->log(sprintf('Loaded %s medias for generating thumbs (provider: %s, context: %s)', count($medias), $provider, $context));
+        $this->log(
+            sprintf(
```
Comment (greg0ire) [inline Command/RefreshMetadataCommand.php]: Please put lines 75 and 76 on the same line, same for 81 and 82

**Comment by vyshkant** [comment]
@greg0ire done!

> Please format your commit message according to our rules ...

Thank you for your patience :)

**Comment by greg0ire** [comment]
Thanks @vyshkant !

**Pull request merged**